packages/Algernon.package/ALGMethodType.class/instance/name.st

```diff
@@ -0,0 +1,4 @@
+accessing
+name
+
+	^ self typeObject asString
```

Comment:
Maybe we could provide this as the default implementation and also delete code in other places?

Reply:
Unfortunately, this is not possible because all other implementations of "name" have a different syntax; the current function of the superclass already covers most of them.

Reply:
Ok.